src/utils/tokens.js

```diff
@@ -10,6 +10,7 @@ import { newTokens, removeTokenMetadata } from '../actions/index';
 import wallet from './wallet';
 import hathorLib from '@hathor/wallet-lib';
 import LOCAL_STORE from '../storage';
+import { getGlobalWallet } from "../modules/wallet";
```

Comment:
```suggestion
import { getGlobalWallet } from '../modules/wallet';
```

Reply:
Can you update the lint file so those start alerting?

Reply:
We really need a linter on this project. This is being tracked on #500